docs/module_docs/featdet_mass_detection/mass-detection.md

```diff
@@ -1,62 +1,47 @@
+:material-menu-open: Raw data methods → Mass detection → Mass detection
+![Mass detection](mass_detection_main.png)
 
-The mass detection steps perform noise filtering (by a threshold) and centroiding of profile raw
-data. The raw data format can either be centroided or in profile mode. If the data is centroided,
-the **centroid** mass detector can be used. Profile data requires a different mass detector such as
-**exact mass**.
+#### **Raw data files**
+Select the input raw data files for the mass detection. All the imported data files can be processed in bulk (_i.e._ _All raw data files_), or a subset can be selected directly from the _MS data files_ panel (_i.e._ _As selected in the main window_) or based on the filename (_i.e._ _File name pattern_) As an alternative, the files' directory can be also specified (_i.e._ _Specific raw data files_). Finally, if the _Those created by previous batch step_ option is selected, MZmine takes the output of the last processing step as input (particularly useful for [batch processing](../../workflows/batch_processing/batch-processing.md)
 
-The mass detection is launched via **Raw data methods → Mass detection → Mass detection** ([1] in
-the figure). In the dropdown menu [2], an applicable mass detector should be chosen and configured
-via the **Setup** button [2]. By selecting the _Show preview_ checkbox, a scan can be selected to
-adjust the noise level.
+#### **Scans**
+Select (or filter out) the MS scans to be processed. Several filters are avialble (_Select filters_ button). A scan number, RT and mobility range can be set (_i.e_ _Scan rnumber_, _Retention time_ and _Mobility_ options); only the scans belonging to the defined range(s) will be processed. The _Base Filtering Integer_ option allows to prcesses one every-_N_ scans. The _Scan definition_ field can be used to filter scans based on the scan's description normally included in the raw file's metadata (_e.g._ FTMS). Scans can also be filtered by _MS level_ (_i.e._ 1, 2, ..., _N_), polarity and spectrum type (_i.e._ Centroided, Profile and Thresholded).
 
-The output of the mass detection step, can be referred to as **mass list**, since it will only
-contain a list of selected m/z values.
+#### **Scan types (IMS)**
+This parameter applies only to IM data and determines if _mobility scans_, _frame scans_ or both (_i.e._ _All scan types_) are processed. For more details about _mobility_ and _frame scans_ see [here](../../terminology/ion-mobility-terminology.md#accumulations-mobility-scans-and-frames) for more details.<br>
+:octicons-light-bulb-16: **Tip**. Since _frame scans_ are obtained by merging multiple _mobility scans_, the noise thresholds will likely be different. However, only one noise level can be set. Therefore, it might be advisable to perform the mass detection only on the _mobility scans_ and select the appropriate noise level. _Mobility scans_ can then be merged into _frame scans_ with a dedicated module (CREATE MODULE DOC). 
```

Comment:
However, only one noise level can be set per processing step. -> Add: If mass detection in frames and mobiltiy scans shall be performed, two module calls are required.

link to mobility scan merging: `[](../featdet_mobility_scan_merging/mobility-scan-merging.md)` (already exists)

Reply:
Better now?